modules/gli_auth0_profile/js/registration.js

```diff
@@ -32,7 +33,10 @@
           drupalSettings.gli_auth0_profile_registration.experience_cloud
         );
       };
-      setTimeout(init, 100);
+      if (!registrationShown) {
```

Comment:
Can we do something like

```js
if (!document.querySelector('#container').hasChildNodes())
```

My fear is that if the script is included more than once there is a chance the variable will get reset.

Reply:
Updated